src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dayPrice = 40;
+  let total = days * dayPrice;
+
+  if (days >= 3 && days < 7) {
+    total = total - 20;
```

Comment:
You can simplify the control flow by using `return` statements inside the `if` blocks directly, thus eliminating the need for an `else` block. This makes the code easier to read and follows the best practice in checklist item 5.

Reply:
I don't understand the comment. There's no 'return' after the first 'if' in the code.